lua/opencode/config.lua

```
@@ -168,6 +168,48 @@ local defaults = {
         end
       end,
     },
+    tmux = {
+      split_window_options = "-h", -- Open in a horizontal split by default
+      ---@param self opencode.provider.Tmux
+      check_tmux = function(self)
+        -- Check if tmux is running in current terminal
+        if not vim.env.TMUX then
+          vim.notify(
+            "opencode.nvim: Tmux provider selected but not running inside a tmux session.",
+            vim.log.levels.ERROR,
+            { title = "opencode.nvim" }
+          )
+          return false
+        end
+        return true
+      end,
```

Remark: alternatively, we can raise an error. I chose this because it's more user-friendly.